### PR TITLE
[Improve](Variant) support bloom filter for variant subcolumns

### PR DESCRIPTION
**Pull request by eldenmoon**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by eldenmoon** [comment]
run buildall

**Comment by eldenmoon** [comment]
run buildall

File: be/src/vec/common/schema_util.cpp [hunk 1]
```diff
@@ -379,10 +379,17 @@ void inherit_tablet_index(TabletSchemaSPtr& schema) {
...
+    for (auto& col : schema->mutable_columns()) {
         if (!col.is_extracted_column()) {
             continue;
         }
+        if (col.type() != FieldType::OLAP_FIELD_TYPE_TINYINT &&
+            col.type() != FieldType::OLAP_FIELD_TYPE_ARRAY &&
+            col.type() != FieldType::OLAP_FIELD_TYPE_DOUBLE &&
+            col.type() != FieldType::OLAP_FIELD_TYPE_FLOAT) {
+            // above types are not supported in bf
+            col.set_is_bf_column(schema->column(col.parent_unique_id()).is_bf_column());
```
Comment (xiaokang) [inline be/src/vec/common/schema_util.cpp:391]: all variant sub columns set bf?

Comment (eldenmoon) [reply be/src/vec/common/schema_util.cpp:391]: yes

**Comment by eldenmoon** [comment]
run buildall

**Review by xiaokang — DISMISSED** [review]
LGTM

**Comment by eldenmoon** [comment]
run buildall

**Comment by eldenmoon** [comment]
run buildall

**Review by qidaye — APPROVED** [review]

**Review by airborne12 — APPROVED** [review]
LGTM

**Pull request merged**